### PR TITLE
Correct North Kalimdor Vendors

### PR DESCRIPTION
**Pull request by NickTyrer**

### CODE REVIEW

File: sql/migrations/20201120223448_world.sql [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 4574 AND `item` = 6365;
+
+-- Missing Items - Elizabeth Van Talen
+INSERT INTO `npc_vendor` (`entry`, `item`, `maxcount`, `incrtime`) VALUES
+(4587, 4389, 1, 10800),
+(4587, 4404, 3, 10800),
+(4587, 4361, 2, 10800);
+
+-- Teldrassil
+-- Missing Items - Felicia Doan
```
Comment (ratkosrb) [inline sql/migrations/20201120223448_world.sql]: Comment says Felicia Doan, but creature id 3608 is Aldia. Which npc is supposed to sell these items?

File: sql/migrations/20201120223448_world.sql [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+
+-- Correct Items - Tiyani
+DELETE FROM `npc_vendor` WHERE `entry` = 4195 AND `item` IN (4536, 4537, 4538, 4539, 4602, 8953, 16171);
+UPDATE `npc_vendor` SET `maxcount` = 0, `incrtime` = 0 WHERE `entry` = 4195 AND `item` IN (21030, 21033, 21031);
+
+-- Correct Items - Allyndia
+DELETE FROM `npc_vendor` WHERE `entry` = 4191 AND `item` IN (4536, 4537, 4538, 4539, 4602, 8953, 16171);
+UPDATE `npc_vendor` SET `maxcount` = 0, `incrtime` = 0 WHERE `entry` = 4191 AND `item` IN (21030, 21033, 21031);
+
+-- Striped Yellowtail - Heldan Galesong
```
Comment (ratkosrb) [inline sql/migrations/20201120223448_world.sql]: Comment says Heldan Galesong, but creature id 4200 is Laird <Fish Vendor>.

File: sql/migrations/20201120223448_world.sql [hunk 3]
```diff
@@ -0,0 +1,431 @@
...
+-- Sharp Throwing Axe - Dalmond
+DELETE FROM `npc_vendor` WHERE `entry` = 4182 AND `item` = 3135;
+
+-- Pattern: Red Woolen Bag - Valdaron
+UPDATE `npc_vendor` SET `maxcount` = 3 WHERE `entry` = 4189 AND `item` = 5772;
+
+-- Pattern: Red Linen Bag - Valdaron
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 4189 AND `item` = 5771;
+
+-- Pattern: Blue Linen Bag - Valdaron
```
Comment (ratkosrb) [inline sql/migrations/20201120223448_world.sql]: Comment says Blue Linen Bag, but item id 6270 is Blue Linen Vest.

File: sql/migrations/20201120223448_world.sql [hunk 4]
```diff
@@ -0,0 +1,431 @@
...
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 7200 WHERE `entry` = 4189 AND `item` = 6270;
+
+-- Ashenvale
+-- Pattern: Herbalist's Gloves - Harloen Darkweave
+UPDATE `npc_vendor` SET `maxcount` = 1, `incrtime` = 10800 WHERE `entry` = 6731 AND `item` = 7361;
+
+-- Correct Items - Maiynn
```
Comment (ratkosrb) [inline sql/migrations/20201120223448_world.sql]: Comment says Maiynn, but creature id 4191 is Allyndia <Food & Drink Vendor>. Which one did you mean to update?

**Comment by NickTyrer** [comment]
@ratkosrb thanks for the review

**Pull request merged**